perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/ServicesManagerMethod.java

```diff
@@ -787,9 +795,17 @@ public Destination call(ApiCaller ac, Deserializer parms) throws PerunException
 		public List<Destination> call(ApiCaller ac, Deserializer parms) throws PerunException {
 			ac.stateChangingCheck();
 
+			Destination destination;
+			
```

Comment:
Implementation is correct, just please double javadoc -> for each existing javadoc add new with "propagationType" param present.